docs/configure-rbac-permissions.md

```diff
@@ -1,70 +1,76 @@
-
-    * Append a Cluster role binding spec:
-
-        ```yaml
-        ---
-        # actual binding to the role
-        kind: ClusterRoleBinding
-        apiVersion: rbac.authorization.k8s.io/v1
-        metadata:
-          name: appinsights-k8s-property-reader-binding
-        subjects:
-        - kind: ServiceAccount
-          name: default
-          namespace: ai-k8s-demo
-        roleRef:
-          kind: ClusterRole
-          name: appinsights-k8s-property-reader
-          apiGroup: rbac.authorization.k8s.io
-        ```
-
-    That is to grant the role of `appinsights-k8s-property-reader` to the default service account in namespace of `ai-k8s-demo`.
-    
+## Setup the permissions for the service account
+
+Depending on various considerations, there could be different strategies to set up the permissions for your service account. Here we list 2 common possibilities, as examples.
+
+* If you want to get the node information along with other info like pod, deployment, and so on, a ClusterRole and a ClusterRoleBinding are required, and here's how to do it:
+
+  * Create a yaml file, [sa-role.yaml](./sa-role.yaml) for example. We will deploy it when it is ready.
+
+      * Write spec to define a cluster role, name it `appinsights-k8s-property-reader` for example:
+   
+          ```yaml
+           kind: ClusterRole
+           apiVersion: rbac.authorization.k8s.io/v1
+           metadata:
+             # "namespace" omitted since ClusterRoles are not namespaced
+             name: appinsights-k8s-property-reader
+           rules:
+           - apiGroups: ["", "apps"]
+             resources: ["pods", "nodes", "replicasets", "deployments"]
+             verbs: ["get", "list"]
+          ```
+          That spec defines the name of the role, and what permission does the role has, for example, list pods.
+
+          You don't have to use the exact name, but you will need to making sure the name is referenced correctly in the following steps.
+
+      * Append a Cluster role binding spec:
+
+          ```yaml
+          ---
+          # actual binding to the role
+          kind: ClusterRoleBinding
+          apiVersion: rbac.authorization.k8s.io/v1
+          metadata:
+            name: appinsights-k8s-property-reader-binding
+          subjects:
+          - kind: ServiceAccount
+            name: default
+            namespace: ai-k8s-demo
+          roleRef:
+            kind: ClusterRole
+            name: appinsights-k8s-property-reader
+            apiGroup: rbac.authorization.k8s.io
+          ```
+
+      That is to grant the role of `appinsights-k8s-property-reader` to the default service account in the namespace of `ai-k8s-demo`.
+
+  * If you don't want to create a Cluster Role, it is also possible to use Role and RoleBinding starting 2.0.6+. Follow the example in [sa-role-none-cluster.yaml](./sa-role-none-cluster.yaml). In that case, you will not have node info on the telemetries.
```

Comment:
What is the `2.0.6` version referring to? Assume that is the version of the ApplicationInsights Kubernetes NuGet package? It would be worth to clarify.